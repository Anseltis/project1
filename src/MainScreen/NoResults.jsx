import React from 'react';

export const NoResults = () => {
    return <h2 className={"no-results__h2"}>No films found</h2>
};