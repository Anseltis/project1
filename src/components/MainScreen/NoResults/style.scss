.no-results {

  &__h2 {
    font-family: Arial, sans-serif;
    font-weight: 600;
    font-size: 1.75rem;
    max-width: 12.5rem;
    margin: 9.375rem auto 0;
  }
}