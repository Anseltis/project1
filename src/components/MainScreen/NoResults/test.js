import React from 'react'
import renderer from 'react-test-renderer'

import { NoResults } from 'components/MainScreen/NoResults/index'

test('NoResult renders properly', () => {
  const wrapper = renderer.create(<NoResults />).toJSON()
  expect(wrapper).toMatchSnapshot()
})
