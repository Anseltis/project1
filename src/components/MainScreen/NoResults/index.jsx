import React from 'react'

import './style.scss'

export const NoResults = () => {
    return <div className='no-result-wrapper'>
    <h2 className='no-results__h2'>No films found</h2>
    </div>
}
