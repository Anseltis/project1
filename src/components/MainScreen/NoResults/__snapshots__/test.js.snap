// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResult renders properly 1`] = `
<h2
  className="no-results__h2"
>
  No films found
</h2>
`;
